### PR TITLE
Add and validate new translations for fr_FR

### PR DESCRIPTION
**Pull request by nils-van-zuijlen**
Also ran `lupdate` to translate the fresh strings

### CODE REVIEW

**Comment by nils-van-zuijlen** [comment]
Ping @jeromelebleu if you want to review

**Comment by mcallegari** [comment]
Let's merge this by now. Thanks :+1:

**Comment by jeromelebleu** [comment]
> Ping @jeromelebleu if you want to review

Thanks for the ping. I didn't have time yet to have a look, sorry about that, merging it was a good choice so!

File: qmlui/qlcplus_fr_FR.ts [hunk 1]
```diff
@@ -62,7 +63,7 @@ Les modifications seront perdues si vous ne les enregistrez pas.</translation>
     <message>
         <location filename="qml/ActionsMenu.qml" line="183"/>
         <source>Open file</source>
-        <translation type="unfinished"></translation>
+        <translation>Ouvrir</translation>
```
Comment (trebmuh) [inline qmlui/qlcplus_fr_FR.ts:66]: Should be "Ouvrir un fichier"

**Pull request merged**